src/docs/root_readme.md

```diff
@@ -1,9 +1,20 @@
-# Usage
-This folder contains a zip file for each device created, named [[device-id]].zip. Transfer each zip to its respective device. A good option for this is to use [scp](https://man7.org/linux/man-pages/man1/scp.1.html).
+# Prerequisites
+Each device must have IoT Edge (must be v1.2 or later) installed. Pick the [supported OS](https://docs.microsoft.com/en-us/azure/iot-edge/support?view=iotedge-2020-11) and follow the [tutorial](https://docs.microsoft.com/en-us/azure/iot-edge/support?view=iotedge-2020-11) to install Azure IoT Edge.
 
-Each device must have iotedge installed. [[TODO: Link iotedge install docs here]]
+Alternatively, if the OS is Tier1 supported OS and using DPS SAS Key provisioning, run [this installation script](https://github.com/Azure/iot-edge-config) can reduce the steps to install and provision.

```

Comment:
The tool doesn't support DPS SAS key provisioning, only Hub SAS and x509

Reply:
Removed the texts completely